src/github/code-scanning-alert.ts

```diff
@@ -20,7 +20,7 @@ export const codeScanningAlertWebhookHandler = async (context: WebhookContext, j
 	}
 
 	context.log.info(`Sending code scanning alert event as Remote Link to Jira: ${jiraClient.baseURL}`);
-	const result = await jiraClient.remoteLink.submit(jiraPayload);
+	const result = await jiraClient.remoteLink.submit(jiraPayload, context.gitHubAppConfig?.gitHubAppId);
```

Comment:
Actually, does it make more sense to put this gitHubAppId into the `jiraClient`, so that we don't change every where that use the method?